tests/PKSim.Tests/ProjectConverter/v9_0/Converter80To90Specs.cs

```diff
@@ -52,4 +54,31 @@ public void should_have_converted_all_population_building_blocks_defined_in_simu
          _allSimulations.Select(x => x.Population).Each(verifyIndividualValueCache);
       }
    }
+
+   public class When_converting_the_population_analysis_74_project_to_90 : ContextWithLoadedProject<Converter80To90>
+   {
+      private List<PopulationSimulation> _allSimulations;
+
+      public override void GlobalContext()
+      {
+         base.GlobalContext();
+         LoadProject("PopulationAnalyses_740");
```

Comment:
@Yuri05 This is the project from #1424. TDD implementation and then I also tried it so pretty sure it works